easybuild/tools/options.py

```diff
@@ -1218,18 +1224,18 @@ def _postprocess_config(self):
         # - the <path> could also specify the location of a *remote* (Git( repository,
         #   which can be done in variety of formats (git@<url>:<org>/<repo>), https://<url>, etc.)
         #   (see also https://github.com/easybuilders/easybuild-framework/issues/3892);
-        path_opt_names = ['buildpath', 'containerpath', 'git_working_dirs_path', 'installpath',
-                          'installpath_modules', 'installpath_software', 'prefix', 'packagepath',
-                          'robot_paths', 'sourcepath']
+        path_opt_names = ['buildpath', 'containerpath', 'failed_installs_build_dirs_path', 'failed_installs_logs_path',
+                          'git_working_dirs_path', 'installpath', 'installpath_modules', 'installpath_software',
+                          'prefix', 'packagepath', 'robot_paths', 'sourcepath']
 
         for opt_name in path_opt_names:
             self._ensure_abs_path(opt_name)
 
         if self.options.prefix is not None:
             # prefix applies to all paths, and repository has to be reinitialised to take new repositorypath in account
             # in the legacy-style configuration, repository is initialised in configuration file itself
-            path_opts = ['buildpath', 'containerpath', 'installpath', 'packagepath', 'repository', 'repositorypath',
-                         'sourcepath']
+            path_opts = ['buildpath', 'containerpath', 'failed_installs_build_dirs_path', 'failed_installs_logs_path',
```

Comment:
@boegel Does this mean the options are always enabled, especially by default? Not sure how `DEFAULT_PATH_SUBDIRS` is taken into account but this part looks like
- if `--prefix` is set
- if `--failed_installs_build_dirs_path` is not set
- then set it to the default subdir under `--prefix`

I wouldn't store those by default, for some installations the build folder is huge (e.g. just NCCL alone) and even the logs can be very large

Reply:
I agree, I still had this in the back of my mind to also change this: using `eb --prefix` shouldn't imply that logs & build dirs are being copied into a subdirectory of the specified prefix.

EasyBuild should be explicitly configured to copy stuff for failing installations, by setting the `failed-installs-build-dirs-path` and/or `--failed-installs-logs-path` configuration options.

I'll add a commit that takes care of that...

Reply:
fixed in 480b92bec

Reply:
fixed in 6a6eef670